docs/topics/migrations.txt

```diff
@@ -47,6 +47,13 @@ Migrations will run the same way on the same dataset and produce consistent
 results, meaning that what you see in development and staging is, under the
 same circumstances, exactly what will happen in production.
 
+.. tip::
+
+    The updates to your application that necessitated migrations should be
+    deployed immediately after deploying the migrations. Otherwise, the
+    changes to your database should be flexible enough to account for the
```

Comment:
How do I make my database flexible as Django created the migration? 🤔

Reply:
Yeah, you're convincing me of wontfix. Good catch, this would need to say something like "Otherwise, avoid making changes to your models that are not compatible with ..."

Reply:
How about just `changes to your database` -> `changes to your models`?

Reply:
> How about just `changes to your database` -> `changes to your models`?

Yes. That could work.

I also wondered if folk who want to run a different version post migration are in the 'you are on you own' category anyway (or should be covered by a more advanced how-to if it is common enough and is something we want to support) 🤔

If so maybe something like your first sentence would suffice? Possibly with an 'its recommended' or an 'its considered best practice'.

Reply:
Yes, I think there are three very good options: zero, one, or two sentences. 🔢

Reply:
😃

Could ask [Guinevere](https://collection.sciencemuseumgroup.org.uk/objects/co8016398/guinevere-random-number-selector-and-name-plaque-lottery-machine-plaque) and set of balls number 6?

Maybe a better bet would be to wait for a fellow to review. 🙂